meson_options.txt

```diff
@@ -1,25 +1,25 @@
 option('tests', type : 'boolean', value : false, description : 'enable tests')
 option('gsettings_desktop_schemas', type : 'feature', value : 'enabled', description : 'enable integration with GNOME desktop preferences')
 option('man', type : 'boolean', value : true, description : 'enable man pages')
-option('packagekit', type : 'boolean', value : true, description : 'enable PackageKit support')
+option('packagekit', type : 'boolean', value : false, description : 'enable PackageKit support')
 option('packagekit_autoremove', type : 'boolean', value : false, description : 'autoremove packages in PackageKit')
 option('polkit', type : 'boolean', value : true, description : 'enable PolKit support')
-option('eos_updater', type : 'boolean', value : false, description : 'enable eos-updater support')
-option('fwupd', type : 'boolean', value : true, description : 'enable fwupd support')
+option('eos_updater', type : 'boolean', value : true, description : 'enable eos-updater support')
+option('fwupd', type : 'boolean', value : false, description : 'enable fwupd support')
 option('flatpak', type : 'boolean', value : true, description : 'enable Flatpak support')
 option('malcontent', type : 'boolean', value : true, description : 'enable parental controls support using libmalcontent')
 option('rpm_ostree', type : 'boolean', value : false, description : 'enable rpm-ostree support')
 option('webapps', type : 'boolean', value : true, description : 'enable webapps support')
 option('hardcoded_foss_webapps', type : 'boolean', value : true, description : 'enable inclusion of a default set of installable FOSS webapps')
 option('hardcoded_proprietary_webapps', type : 'boolean', value : true, description : 'enable inclusion of a default set of installable proprietary webapps')
```

Comment:
I don't think we want this one to be `true`, do we flip it to False in a future patch?

Reply:
This looks the same as we have for [meson_options.txt in eos5.1](https://github.com/endlessm/gnome-software/blob/eos5.1/meson_options.txt) :thinking:

Reply:
It is flipped to false in `debian/rules`, which is used on OBS but not Jenkins. https://github.com/endlessm/gnome-software/commit/0e8949841daab47f395548885757baafbe63d50a

Reply:
Oh, that line! Github's had me looking at line 7 instead :b I see we did that in https://phabricator.endlessm.com/T34569. It does seem a bit odd, so I went ahead and set the option here while we have the opportunity.